lark/load_grammar.py

```diff
@@ -1072,7 +1082,7 @@ def _unpack_definition(self, tree, mangle):
         return name, exp, params, opts
 
 
-    def load_grammar(self, grammar_text: str, grammar_name: str="<?>", mangle: Callable[[str], str]=None) -> None:
+    def load_grammar(self, grammar_text: str, grammar_name: str="<?>", mangle: Optional[Callable[[str], str]]=None) -> None:
```

Comment:
Maybe I am wrong here, but isn't `Optional` automatically added when `=None` is the default?

Reply:
Interesting, I didn't know about that feature. However, it looks like its use is no longer recommended. [https://stackoverflow.com/questions/62732402/can-i-omit-optional-if-i-set-default-to-none](https://stackoverflow.com/questions/62732402/can-i-omit-optional-if-i-set-default-to-none)

Reply:
Aha. I didn't see that it was deprecated. Good to know.